test/TEST-13-NSPAWN-SMOKE/test.sh

```diff
@@ -85,26 +84,29 @@ function run {
     fi
     if [[ "$2" = "yes" && "$is_cgns_supported" = "no" ]];  then
         printf "Cgroup namespaces are not supported. Skipping.\n"
+        return
     fi
 
-    export UNIFIED_CGROUP_HIERARCHY="$1"
-    export SYSTEMD_NSPAWN_USE_CGNS="$2"
-    local _root="/var/lib/machines/unified-$1-cgns-$2"
-    /create-busybox-container "$_root"
-    systemd-nspawn --register=no -D "$_root" -b
-    systemd-nspawn --register=no -D "$_root" --private-network -b
-    systemd-nspawn --register=no -D "$_root" -U -b
-    systemd-nspawn --register=no -D "$_root" --private-network -U -b
-    unset UNIFIED_CGROUP_HIERARCHY
-    unset SYSTEMD_NSPAWN_USE_CGNS
-}
+    set -x
+    # Call sleep before each container to make the logs easier to parse
+    systemd-run -E UNIFIED_CGROUP_HIERARCHY=$1 -E SYSTEMD_NSPAWN_USE_CGNS=$2 -E SYSTEMD_LOG_LEVEL=debug \
+                --wait systemd-nspawn --quiet --register=no -M container -b
```

Comment:
Why? Actually, this `-x` helps to understand what is going on.
I guess we can
```
UNIFIED_CGROUP_HIERARCHY=$1 SYSTEMD_NSPAWN_USE_CGNS=$2 systemd-nspawn --register=no -D "$_root" -b
```
instead of the
```
export ..
export ..
systemd-nspawn
unset
unset
...
```

This fixes:
> When anything fails, it is very hard to locate the failure, when everything
is run from the same script one after another.

No?